imap/imapd.c

```diff
@@ -13325,6 +13326,34 @@ static void add_intermediates(const char *extname, struct list_rock *lrock)
     strarray_fini(&inter);
 }
 
+static int is_noinferiors(struct findall_data *data, struct list_rock *rock)
+{
+    int r = 0;
+
+    if (data->mb_category == MBNAME_ALTINBOX)
```

Comment:
nit: I suggest to also add block-braces here, or handle the else condition first, followed by an else with the single statement. What we really should to do is agree on what to do in that case in the style guide. Note that the style guide already forbids an empty line between if and else blocks.

Reply:
Fixed